Resources/Prototypes/Roles/Jobs/Security/detective.yml

```diff
@@ -7,12 +7,14 @@
   - !type:DepartmentTimeRequirement
     department: Security
     time: 54000 # 15 hours
+  - !type:DepartmentTimeRequirement
+      department: Civilian
+      time: 18000 # 5 hours
   startingGear: DetectiveGear
   icon: "JobIconDetective"
-  supervisors: job-supervisors-hos
+  supervisors: job-supervisors-security-hire
   canBeAntag: false

```

Comment:
if it's not a sec role then it probably should be allowed to be an antag and not have mindshield, just like lawyer

Reply:
This will likely result in cutting out most of the gameplay for the detective.

Reply:
this will also give them a free gun able to shot the 0.45 AP ammo, resulting in 8 TC economy if they ever decide to go loud with their revolver